LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -2205,6 +2205,10 @@ RewardDetails="You'll get a selection of random enemy corpses."
 
 [Reward_Dummy_StatBoost X2RewardTemplate]
 DisplayName="Stat Boosts"
+
+[Reward_ResistanceMec X2RewardTemplate]
+DisplayName="Resistance MEC"
+RewardDetails="You'll get a Resistance MEC to guard one of your Havens."
```

Comment:
There's already a localisation entry for this reward. Add the reward details to that one please.